pwndbg/gdblib/disasm/aarch64.py

```diff
@@ -132,5 +168,62 @@ def _set_annotation_string(self, instruction: PwndbgInstruction, emu: Emulator)
         # Dispatch to the correct handler
         self.annotation_handlers.get(instruction.id, lambda *a: None)(instruction, emu)
 
+    def _register_width(self, instruction: PwndbgInstruction, op: EnhancedOperand) -> int:
+        # TODO: find a better way to do this. Cache this on the operand object?
+        return 32 if instruction.cs_insn.reg_name(op.reg).upper().startswith("W") else 64
```

Comment:
Can the reg name be 'w' or 'W'? Aren't all lowercase or uppercase? Maybe we can skip the `.upper()` part?

Also: `s[0] == 'W'` is probably faster than `s.startswith("W")`.

Reply:
Made this change